elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
+  (and (not (= (length elixir-format-mix-path) 0)) (not (= (length elixir-format-elixir-path) 0)))
```

Comment:
This will only return true if if both are unset, and if we're going to be doing a single check, it should return true if either of them are unset -- which means we can just do `(or (= (length elixir-format-mix-path) 0) (= (length elixir-format-elixir-path) 0)`.

Reply:
done.